This ensures that efficient capability checks are used on musl.

--- a/fat-ppc.c
+++ b/fat-ppc.c
@@ -42,12 +42,10 @@
 
 #if defined(_AIX)
 # include <sys/systemcfg.h>
-#elif defined(__linux__) && defined(__GLIBC__) && defined(__GLIBC_PREREQ)
-# if __GLIBC_PREREQ(2, 16)
-#  define USE_GETAUXVAL 1
-#  include <asm/cputable.h>
-#  include <sys/auxv.h>
-# endif
+#elif defined(__linux__)
+# define USE_GETAUXVAL 1
+# include <asm/cputable.h>
+# include <sys/auxv.h>
 #elif defined(__FreeBSD__)
 # include <machine/cpu.h>
 # ifdef PPC_FEATURE2_HAS_VEC_CRYPTO
